modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js

```diff
@@ -55,13 +55,26 @@ const defaultProps = {
   threshold: {type: 'number', min: 0, max: 1, value: 0.05}
 };
 
+const REQUIRED_FEATURES = [
+  FEATURES.WEBGL2, // TODO: Remove after trannsform refactor
+  FEATURES.COLOR_ATTACHMENT_RGBA32F, // weight-map generation
+  FEATURES.BLEND_EQUATION_MINMAX, // max weight calculation
+  FEATURES.FLOAT_BLEND, // weight-map generation and max weight calculation
+  FEATURES.TEXTURE_FLOAT // weight-map as texture
+];
+
 export default class HeatmapLayer extends CompositeLayer {
   initializeState() {
     const {gl} = this.context;
     const textureSize = Math.min(SIZE_2K, getParameter(gl, gl.MAX_TEXTURE_SIZE));
     this.state = {textureSize, supported: true};
-    if (!isWebGL2(gl)) {
-      log.error(`HeatmapLayer ${this.id} is not supported on this browser, requires WebGL2`)();
+    if (!hasFeatures(gl, REQUIRED_FEATURES)) {
+      const info = REQUIRED_FEATURES.reduce(
```

Comment:
Call `GPUAggregator.isSupported`?

Reply:
Though the checks are same, HeatmapLayer doesn't use GPUGridAggregator, in future it is possible these feature list will change.